lib/tests/option.rs

```diff
@@ -0,0 +1,48 @@
+mod parse;
+use parse::Parse;
+mod helpers;
+use helpers::new_ds;
+use surrealdb::dbs::Session;
+use surrealdb::err::Error;
+use surrealdb::sql::Value;
+
+#[tokio::test]
+async fn option_import_indexes_should_be_populated() -> Result<(), Error> {
+	let sql = "
+		OPTION IMPORT;
+		DEFINE INDEX field_num ON test FIELDS num;
+		CREATE ONLY test:1 SET num = 123;
+		SELECT * FROM test WHERE num = 123;
```

Comment:
Could you also add a test checking the output of this:

`SELECT * FROM test WHERE num = 123 EXPLAIN;`

This will ensure the index has been created and used. Avoiding any additional "bypass" the import option would do

Reply:
Good point, thanks!